includes/parser/Parser.php

```diff
@@ -5423,7 +5423,7 @@ function makeImage( $title, $options, $holders = false ) {
 			'horizAlign' => array(), 'vertAlign' => array() );
 
 		# wikia start
-		if (!wfRunHooks( 'BeforeParserMakeImageLinkObjOptions', array( &$this, &$title, &$parts, &$params, &$time, &$descQuery, $options ) ) ) {
+		if (!wfRunHooks( 'BeforeParserMakeImageLinkObjOptions', array( $this, &$title, &$parts, &$params, &$time, &$descQuery, $options ) ) ) {
 			$ret = Linker::makeImageLink2( $title, $file, $params['frame'], $params['handler'], $time, $descQuery );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$ret". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)